19-10-2020/main.js

```diff
@@ -0,0 +1,133 @@
+// Bài 1
+// Viết một chương trình nhập lương nhân viên, tính thuế thu nhập và lương ròng(số tiền lương thực sự mà nhân viên đó nhận được).Với các thông số giả sử như sau(không theo luật lương, chỉ là con số giả sử để dễ tính toán):
+// 30 % thuế thu nhập nếu lương là 15 triệu.
+// 20 % thuế thu nhập nếu lương từ 7 đến 15 triệu.
+// 10 % thuế thu nhập nếu lương dưới 7 triệu.
+const getSalary = (salary) => {
+    if (salary >= 15000000) {
+        console.log('Thuế thu nhập', salary * 0.3)
+        console.log('Lương ròng   ', salary * 0.7)
+    } else if (salary >= 7000000) {
+        console.log('Thuế thu nhập', salary * 0.2)
+        console.log('Lương ròng   ', salary * 0.8)
+    } else {
+        console.log('Thuế thu nhập', salary * 0.1)
+        console.log('Lương ròng   ', salary * 0.9)
```

Comment:
```suggestion
        console.log('Lương ròng', salary * 0.9)
```
Nhà thừa dấu cách hay gì

Reply:
Cái này là em căn cho đều nhau mà ==